tools/pylib/boututils/surface_average.py

```diff
@@ -35,37 +34,28 @@ def surface_average(var, g, area=None):
 
     s = np.ndim(var)
 
-
-
     if s == 4 :
         nx = np.shape(var)[1]
         ny = np.shape(var)[2]
         nt = np.shape(var)[0]
 
         result = np.zeros((nx,nt))
         for t in range (nt):
-
-            result[:,t] = surface_average(var[t,:,:,:], g, area=area)
+            result[:,t] = surface_average(var[t,:,:,:], grid, area=area)
```

Comment:
I'd prefer to switch the result so it's `result[t,x]`.

Reply:
I would rather remove it ... If you think this function is useful I'd do it though ...